### DISCUSSION TITLE
TM1py Community Blog: TM1py Tales

### DISCUSSION THREAD

**Discussion by MariusWirtz** [cubewise-code/tm1py#581] [General]
Hi everyone,

I created a new community blog for TM1py.
The "TM1py Tales" blog is a GitHub project for the community to share ideas and stories.
I will regularly share blueprints and ideas through this platform as well.

We are leveraging GitHub Pages, to build a blog based on the content in the GitHub repository.
It's a wonderful technology. When people add mark-down files to the GitHub project the blog updates automagically 🤯

https://github.com/cubewise-code/tm1py-tales

https://cubewise-code.github.io/tm1py-tales/

I wrote the first post today about the different writeback functions available in TM1py.

To anyone that wants to share an idea or a use case re TM1py, please feel free to add a little blog post as a markdown file into the [_posts](https://github.com/cubewise-code/tm1py-tales/tree/master/_posts) folder on GitHub through a Pull Request.

I hope over time this turns into a kind of knowledge repository for TM1py.

**Comment by wimgielis** [comment]
When time permits I can write a blog post on how to keep track of
consulting and other activities in your Outlook agenda, and having invoices
rolling out automatically with TM1 and tm1py 😃





Op wo 28 jul. 2021 om 20:02 schreef Marius Wirtz ***@***.***>

> Hi everyone,
>
> I created a new community blog for TM1py.
> The "TM1py Tales" blog is a GitHub project for the community to share
> ideas and stories.
> I will regularly share blueprints and ideas through this platform as well.
>
> We are leveraging GitHub Pages, to build a blog based on the content in
> the GitHub repository.
> It's a wonderful technology. When people add mark-down files to the GitHub
> project the blog updates automagically 🤯
>
> https://github.com/cubewise-code/tm1py-tales
>
> https://cubewise-code.github.io/tm1py-tales/
>
> I wrote the first post today about the different writeback functions
> available in TM1py.
>
> To anyone that wants to share an idea or a use case re TM1py, please feel
> free to add a little blog post as a markdown file into the _posts
> <https://github.com/cubewise-code/tm1py-tales/tree/master/_posts> folder
> on GitHub through a Pull Request.
>
> I hope over time this turns into a kind of knowledge repository for TM1py.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/cubewise-code/tm1py/discussions/581>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AEDHULODYBKYNX5PSMMPLZTT2BA45ANCNFSM5BE3KNJQ>
> .
>
-- 

------

Best regards / Beste groeten,

Wim Gielis
MS Excel MVP 2011-2014
https://www.wimgielis.com <http://www.wimgielis.be>
